app/src/pages/playground/ModelComboBox.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useMemo, useState } from "react";
+import { graphql, useFragment } from "react-relay";
+
+import { ComboBox, ComboBoxItem, ComboBoxProps } from "@phoenix/components";
+
+import { ModelComboBoxFragment$key } from "./__generated__/ModelComboBoxFragment.graphql";
+
+type ModelComboBoxProps = {
+  query: ModelComboBoxFragment$key;
+  onChange: (model: string) => void;
+  provider: ModelProvider;
+  modelName: string | null;
+  container?: HTMLElement;
+} & Omit<
+  ComboBoxProps<{ name: string }>,
+  "children" | "onSelectionChange" | "defaultSelectedKey"
+>;
+
+export function ModelComboBox({
+  query,
+  onChange,
+  container,
+  ...props
+}: ModelComboBoxProps) {
+  const data = useFragment<ModelComboBoxFragment$key>(
+    graphql`
+      fragment ModelComboBoxFragment on Query
+      @argumentDefinitions(
+        providerKey: { type: "GenerativeProviderKey!", defaultValue: OPENAI }
+      ) {
+        models(input: { providerKey: $providerKey }) {
+          name
+        }
+      }
+    `,
+    query
+  );
+
+  const [modelInput, setModelInput] = useState("");
```

Comment:
should have dont git mv before editing this file, sorry! basicaly just from here down has changed, render has not changed much since picker and combobox are pretty similar